modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/header/page.jsp

```diff
@@ -328,7 +328,7 @@ String myWorkflowTasksPortletNamespace = PortalUtil.getPortletNamespace(PortletK
 
 					<c:if test="<%= Validator.isNotNull(previewUrl) %>">
 						<clay:link
-							elementClasses="btn btn-outline-borderless btn-outline-secondary btn-sm text-primary"
+							cssClass="btn btn-outline-borderless btn-outline-secondary btn-sm text-primary"
```

Comment:
I think many of these can be accomplished using the API with `type="button"` and `displayType`, `borderless` and `size` rather than custom css classes.